storage/innobase/xtrabackup/test/t/pxb-1961.sh

```diff
@@ -0,0 +1,65 @@
+############################################################################
+# PXB-1961: xtraback fails with encryption='n' with Percona server
+############################################################################
+
+function backup_and_restore() {
+record_db_state test
+rm -rf $topdir/backup
+mkdir -p $topdir/backup
+# Backup
+xtrabackup --datadir=$mysql_datadir --backup --target-dir=$topdir/backup 
+# Prepare
+xtrabackup --datadir=$mysql_datadir --prepare --target-dir=$topdir/backup
+stop_server
+# Restore
+rm -rf $mysql_datadir/ibdata1 $mysql_datadir/ib_logfile* \
+    $mysql_datadir/test/*.ibd
+cp -r $topdir/backup/* $mysql_datadir
+start_server
+# Verify backup
+verify_db_state test
+
+}
+if ! is_debug_server  ; then
+       skip_test "Require debug server version"
+fi
+
+. inc/common.sh
```

Comment:
Don't we need `. inc/common.sh` before the  `if ! is_debug_server` for `is_debug_server`  function to be available?